.gitignore

```diff
@@ -1,3 +1,12 @@
 /lib/assets/node_modules
 tmp/
 coverage.txt
+
```

Comment:
these files should be part of your global .gitignore. generally ide/system items shouldn't be defined for each project

Reply:
Goland is one of the hot GO development tools, and these ignore lists are necessary

Reply:
Do you know what a global .gitignore is? 
**IDE and System items should not be defined seperately for each project**.

Reply:
If you open a project using Goland, the `.ide`a folder will be generated under the root of the project

Reply:
I know what the .idea folder is. I use Goland as well. But that shouldn't stop this project from following these guidelines.